### PR TITLE
Create container-orchestration.md

### PR DESCRIPTION
**Pull request by Aryan-Singla**
Signed off by : Aryan-Singla [devendergupta4573@gmail.com]

### Describe your changes
Localize container-orchestration.md to Hindi

### Related issue number or link (ex: `resolves #issue-number`)
resolves https://github.com/cncf/glossary/issues/1977

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by ARYANis0001** [comment]
Please assign this issue to me I have raised the PR

**Review by ARYANis0001 — COMMENTED** [review]
Updated

File: content/hi/container-orchestration.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+## यह क्या है
+
+[कंटेनर](/कंटेनर/) ऑर्केस्ट्रेशन का तात्पर्य गतिशील वातावरण में कंटेनरीकृत अनुप्रयोगों के जीवनचक्र को प्रबंधित और स्वचालित करने से है।
+इसे एक कंटेनर ऑर्केस्ट्रेटर (ज्यादातर मामलों में, [कुबेरनेट्स](/कुबेरनेट्स)) के माध्यम से निष्पादित किया जाता है, जो तैनाती, (ऑटो) स्केलिंग, ऑटो-हीलिंग और मॉनिटरिंग को सक्षम बनाता है।
+आर्केस्ट्रा एक रूपक है:
+ऑर्केस्ट्रेशन टूल एक संगीत कंडक्टर की तरह कंटेनरों का संचालन करता है, यह सुनिश्चित करता है कि प्रत्येक कंटेनर (या संगीतकार) वही करे जो उसे करना चाहिए।
+
+## यह समस्या का समाधान करता है
```
Comment (kumarankit999) [inline content/hi/container-orchestration.md:14]: ## यह किस समस्या का समाधान करता है

**Comment by Aryan-Singla** [comment]
Hey @aj11anuj @kumarankit999  please review it

**Review by ankur0904 — CHANGES_REQUESTED** [review]
Please have a look.

**Comment by Aryan-Singla** [comment]
@ankur0904 I have added your suggestions

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

**Review by Garima-Negi — APPROVED** [review]

**Comment by Aryan-Singla** [comment]
Hey @kumarankit999 if you find this PR is correct. Can you please approve it. Already received 1 approval

File: content/hi/container-orchestration.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+[माइक्रोसर्विसेज](/माइक्रोसर्विसेज/), सुरक्षा और नेटवर्क संचार को बड़े पैमाने पर प्रबंधित करना - और [वितरित सिस्टम](/वितरित-सिस्टम/) को सामान्य रूप से प्रबंधित करना - मैन्युअल रूप से प्रबंधित करना असंभव नहीं लेकिन कठिन है।
+कंटेनर ऑर्केस्ट्रेशन उपयोगकर्ताओं को इन सभी प्रबंधन कार्यों को स्वचालित करने की अनुमति देता है।
+
+## यह कैसे मदद करता है
+
+कंटेनर ऑर्केस्ट्रेशन उपकरण उपयोगकर्ताओं को सिस्टम की स्थिति निर्धारित करने की अनुमति देते हैं।
+सबसे पहले, वे घोषणा करते हैं कि यह कैसा दिखना चाहिए (उदाहरण के लिए, एक्स कंटेनर, वाई पॉड्स, आदि)।
```
Comment (ankur0904) [inline content/hi/container-orchestration.md:22]:
```suggestion
सबसे पहले, वे परिभाषित करते हैं कि यह कैसा दिखना चाहिए (उदाहरण के लिए X कंटेनर, Y पॉड्स, आदि)।
```

File: content/hi/container-orchestration.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ऑर्केस्ट्रेशन टूल तब स्वचालित रूप से बुनियादी ढांचे की निगरानी करेगा और यदि इसकी स्थिति घोषित स्थिति से भटकती है तो इसे ठीक कर देगा (उदाहरण के लिए, यदि कोई कंटेनर दुर्घटनाग्रस्त हो जाता है तो एक नया कंटेनर स्पिन करें)।
```
Comment (ankur0904) [inline content/hi/container-orchestration.md:23]:
```suggestion
ऑर्केस्ट्रेशन टूल तब स्वचालित रूप से बुनियादी ढांचे की निगरानी करेगा और यदि इसकी स्थिति परिभाषित स्थिति से भटकती है तो इसे ठीक कर देगा (उदाहरण के लिए यदि कोई कंटेनर दुर्घटनाग्रस्त हो जाता है तो एक नया कंटेनर स्पिन करें)।
```

**Review by ankur0904 — COMMENTED** [review]
Please *don't* consider the change until other maintainers approves the suggested changes.

_Thanks_

**Review by abhay-raj19 — APPROVED** [review]
Looks good to me !

**Comment by bishal7679** [comment]
Please wait for final approval!

**Comment by Aryan-Singla** [comment]
@bishal7679 Please check slack glossary-loclization-hindi

**Review by kumarankit999 — COMMENTED** [review]
lgtm

**Review by ankur0904 — APPROVED** [review]
LGTM

**Comment by aj11anuj** [comment]
> Please _don't_ consider the change until other maintainers approves the suggested changes.
> 
> _Thanks_

@ankur0904 feel free to review Hindi localization PR's, no need to wait for our approval for your reviews to be considered helpful. 
@Aryan-Singla, you can consider reviews from other people as well and implement changes if you feel like it. We will take care of unhelpful changes (if there are any) during final review and approval.

File: content/hi/container-orchestration.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+## यह क्या है
+
+[कंटेनर](/कंटेनर/) ऑर्केस्ट्रेशन का तात्पर्य गतिशील वातावरण में कंटेनरीकृत अनुप्रयोगों के जीवनचक्र को प्रबंधित और स्वचालित करने से है।
+इसे एक कंटेनर ऑर्केस्ट्रेटर (ज्यादातर मामलों में, [कुबेरनेट्स](/कुबेरनेट्स/)) के माध्यम से निष्पादित किया जाता है, जो तैनाती, (ऑटो) स्केलिंग, ऑटो-हीलिंग और मॉनिटरिंग को सक्षम बनाता है।
+आर्केस्ट्रा एक रूपक है:
+ऑर्केस्ट्रेशन टूल एक संगीत कंडक्टर की तरह कंटेनरों का संचालन करता है, यह सुनिश्चित करता है कि प्रत्येक कंटेनर (या संगीतकार) वही करे जो उसे करना चाहिए।
+
+## यह किस समस्या का समाधान करता है
+
+[माइक्रोसर्विसेज](/माइक्रोसर्विसेज/), सुरक्षा और नेटवर्क संचार को बड़े पैमाने पर प्रबंधित करना - और [वितरित सिस्टम](/वितरित-सिस्टम/) को सामान्य रूप से प्रबंधित करना - मैन्युअल रूप से प्रबंधित करना असंभव नहीं लेकिन कठिन है।
```
Comment (bishal7679) [inline content/hi/container-orchestration.md:16]:
```suggestion
[माइक्रोसर्विसेज](/microservices/), सुरक्षा और नेटवर्क संचार को बड़े पैमाने पर प्रबंधित करना - और [वितरित सिस्टम](/distributed-systems/) को सामान्य रूप से प्रबंधित करना - मैन्युअल रूप से प्रबंधित करना असंभव नहीं लेकिन कठिन है।
```

File: content/hi/container-orchestration.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+कंटेनर ऑर्केस्ट्रेशन उपयोगकर्ताओं को इन सभी प्रबंधन कार्यों को स्वचालित करने की अनुमति देता है।
+
+## यह कैसे मदद करता है
```
Comment (bishal7679) [inline content/hi/container-orchestration.md:19]:
```suggestion
## समाधान
```

File: content/hi/container-orchestration.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+कंटेनर ऑर्केस्ट्रेशन उपकरण उपयोगकर्ताओं को सिस्टम की स्थिति निर्धारित करने की अनुमति देते हैं।
```
Comment (bishal7679) [inline content/hi/container-orchestration.md:21]:
```suggestion
कंटेनर ऑर्केस्ट्रेशन टूल उपयोगकर्ताओं को सिस्टम की स्थिति निर्धारित करने की अनुमति देते हैं।
```

File: content/hi/container-orchestration.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+सबसे पहले, वे परिभाषित करते हैं कि यह कैसा दिखना चाहिए (उदाहरण के लिए X कंटेनर, Y पॉड्स, आदि)।
+ऑर्केस्ट्रेशन टूल तब स्वचालित रूप से बुनियादी ढांचे की निगरानी करेगा और यदि इसकी स्थिति परिभाषित स्थिति से भटकती है तो इसे ठीक कर देगा (उदाहरण के लिए यदि कोई कंटेनर दुर्घटनाग्रस्त हो जाता है तो एक नया कंटेनर स्पिन करें)।
```
Comment (bishal7679) [inline content/hi/container-orchestration.md:23]:
```suggestion
ऑर्केस्ट्रेशन टूल तब स्वचालित रूप से इंफ्रास्ट्रक्चर की निगरानी करेगा और यदि इसकी स्थिति परिभाषित स्थिति से भटकती है तो इसे ठीक कर देगा (उदाहरण के लिए यदि कोई कंटेनर दुर्घटनाग्रस्त हो जाता है तो एक नया कंटेनर स्पिन करें)।
```

**Comment by Aryan-Singla** [comment]
@kumarankit999  assign issue(#1977) to me. We have been working on it for the last few days but the issue is not assigned to me

File: content/hi/container-orchestration.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+यह स्वचालन कई इंजीनियरिंग टीमों के अन्यथा अत्यधिक मैन्युअल और जटिल परिचालन कार्यों को सरल बनाता है, जिसमें प्रावधान, तैनाती, स्केलिंग (ऊपर और नीचे), नेटवर्किंग, लोड संतुलन और अन्य गतिविधियां शामिल हैं।
```
Comment (bishal7679) [inline content/hi/container-orchestration.md:24]:
```suggestion
यह स्वचालन कई इंजीनियरिंग टीमों के अन्यथा अत्यधिक मैन्युअल और जटिल परिचालन कार्यों को सरल बनाता है, जिसमें प्रावधान, डिप्लॉयमेंट, स्केलिंग (उप और डाउन), नेटवर्किंग, लोड संतुलन और अन्य गतिविधियां शामिल हैं।
```

**Review by bishal7679 — CHANGES_REQUESTED** [review]
Left some suggestions!

**Review by bishal7679 — APPROVED** [review]
Thank you @Aryan-Singla!

**Pull request merged**